src/dispatch/plugins/dispatch_slack/case/interactive.py

```diff
@@ -0,0 +1,493 @@
+        db_session=db_session,
+        project_id=project_id,
+    )
+
+    blocks = [
+        Context(elements=[MarkdownText(text="Accept the defaults or adjust as needed.")]),
+        title_input(),
+        description_input(),
+        project_select(
+            db_session=db_session,
+            initial_option={"text": project.name, "value": project.id},
+            action_id=CaseEscalateActions.project_select,
+            dispatch_action=True,
+        ),
+        case_type_select(
+            db_session=db_session, initial_option=None, project_id=project.id, block_id=None
+        ),
+        case_priority_select(
+            db_session=db_session,
+            project_id=project.id,
+            initial_option=None,
+            optional=True,
+            block_id=None,  # ensures state is reset
+        ),
+    ]
+
+    modal = Modal(
+        title="Report Issue",
+        blocks=blocks,
+        submit="Report",
+        close="Close",
+        callback_id=CaseReportActions.submit,
+        private_metadata=context["subject"].json(),
+    ).build()
+
+    await client.views_update(
+        view_id=body["view"]["id"],
+        hash=body["view"]["hash"],
+        trigger_id=body["trigger_id"],
+        view=modal,
+    )
+
+
+@app.view(
+    CaseReportActions.submit,
+    middleware=[db_middleware, action_context_middleware, modal_submit_middleware, user_middleware],
+)
+async def handle_report_submission_event(
+    ack, body, context, form_data, db_session, user, client, logger
+):
+    await ack()
+
+    case_priority = None
+    if form_data.get(DefaultBlockIds.case_priority_select):
+        case_priority = {"name": form_data[DefaultBlockIds.case_priority_select]["name"]}
+
+    case_type = None
+    if form_data.get(DefaultBlockIds.case_type_select):
+        case_type = {"name": form_data[DefaultBlockIds.case_type_select]["name"]}
+
+    case_in = CaseCreate(
+        title=form_data[DefaultBlockIds.title_input],
+        description=form_data[DefaultBlockIds.description_input],
+        status=CaseStatus.new,
+        case_priority=case_priority,
+        case_type=case_type,
+    )
+
+    case = case_service.create(db_session=db_session, case_in=case_in, current_user=user)
+
+    modal = Modal(
+        title="Case Created",
+        close="Close",
+        blocks=[Section(text="Your case has been created. Running case execution flows now...")],
+    ).build()
+
+    await client.views_update(
+        view_id=body["view"]["id"],
+        hash=body["view"]["hash"],
+        trigger_id=body["trigger_id"],
+        view=modal,
+    )
+
+    case_flows.case_new_create_flow(
+        case_id=case.id, organization_slug=context["subject"].organization_slug
+    )
+
+    modal = Modal(
+        title="Case Created",
+        close="Close",
+        blocks=[Section(text="Your case has been created.")],
+    ).build()
+
+    await client.views_update(
+        view_id=body["view"]["id"],
+        hash=body["view"]["hash"],
```

Comment:
I could use a sanity check on why this works, since it seems backwards (removing hash verification to resolve the hash conflict).

```suggestion
```

```bash
ERROR:The request to the Slack API failed. (url: https://www.slack.com/api/views.update)
The server responded with: {'ok': False, 'error': 'hash_conflict', 'view': {'id': 'V04K54REAQP', 'team_id': 'T04FZTFLBFE', 'type': 'modal', 'blocks': [{'type': 'section', 'block_id': 'nQZl', 'text': {'type': 'mrkdwn', 'text': 'Your case has been created. Running case execution flows now...', 'verbatim': False}}], 'private_metadata': '', 'callback_id': '', 'state': {'values': {}}, 'hash': '1674104238.a4CyhoGD', 'title': {'type': 'plain_text', 'text': 'Case Created', 'emoji': True}, 'clear_on_close': False, 'notify_on_close': False, 'close': {'type': 'plain_text', 'text': 'Close', 'emoji': True}, 'submit': None, 'previous_view_id': None, 'root_view_id': 'V04K54REAQP', 'app_id': 'A04FGTKNP2B', 'external_id': '', 'app_installed_team_id': 'T04FZTFLBFE', 'bot_id': 'B04FXFFMSFL'}}:/Users/wshel/Projects/dispatch/src/dispatch/plugins/dispatch_slack/bolt.py:build_and_log_error:102
Traceback (most recent call last):
  File "/Users/wshel/.pyenv/versions/3.9.7/envs/dispatch/lib/python3.9/site-packages/slack_bolt/listener/asyncio_runner.py", line 65, in run
    returned_value = await listener.run_ack_function(request=request, response=response)
  File "/Users/wshel/.pyenv/versions/3.9.7/envs/dispatch/lib/python3.9/site-packages/slack_bolt/listener/async_listener.py", line 119, in run_ack_function
    return await self.ack_function(
  File "/Users/wshel/Projects/dispatch/src/dispatch/plugins/dispatch_slack/case/interactive.py", line 488, in handle_report_submission_event
    await client.views_update(
  File "/Users/wshel/.pyenv/versions/3.9.7/envs/dispatch/lib/python3.9/site-packages/slack_sdk/web/async_client.py", line 4519, in views_update
    return await self.api_call("views.update", json=kwargs)
  File "/Users/wshel/.pyenv/versions/3.9.7/envs/dispatch/lib/python3.9/site-packages/slack_sdk/web/async_base_client.py", line 161, in api_call
    return await self._send(
  File "/Users/wshel/.pyenv/versions/3.9.7/envs/dispatch/lib/python3.9/site-packages/slack_sdk/web/async_base_client.py", line 201, in _send
    return AsyncSlackResponse(**{**data, **res}).validate()
  File "/Users/wshel/.pyenv/versions/3.9.7/envs/dispatch/lib/python3.9/site-packages/slack_sdk/web/async_slack_response.py", line 203, in validate
    raise e.SlackApiError(message=msg, response=self)
slack_sdk.errors.SlackApiError: The request to the Slack API failed. (url: https://www.slack.com/api/views.update)
The server responded with: {'ok': False, 'error': 'hash_conflict', 'view': {'id': 'V04K54REAQP', 'team_id': 'T04FZTFLBFE', 'type': 'modal', 'blocks': [{'type': 'section', 'block_id': 'nQZl', 'text': {'type': 'mrkdwn', 'text': 'Your case has been created. Running case execution flows now...', 'verbatim': False}}], 'private_metadata': '', 'callback_id': '', 'state': {'values': {}}, 'hash': '1674104238.a4CyhoGD', 'title': {'type': 'plain_text', 'text': 'Case Created', 'emoji': True}, 'clear_on_close': False, 'notify_on_close': False, 'close': {'type': 'plain_text', 'text': 'Close', 'emoji': True}, 'submit': None, 'previous_view_id': None, 'root_view_id': 'V04K54REAQP', 'app_id': 'A04FGTKNP2B', 'external_id': '', 'app_installed_team_id': 'T04FZTFLBFE', 'bot_id': 'B04FXFFMSFL'}}
```